src/tests/system/tests/test_logging.py

```diff
@@ -100,64 +158,107 @@ def test_logging__default_settings_logs_domain_configuration_errors(client: Clie
 
 @pytest.mark.integration
 @pytest.mark.importance("low")
-@pytest.mark.ticket(bz=1893159)
 @pytest.mark.topology(KnownTopology.LDAP)
-def test_logging__default_settings_logs_offline_errors(client: Client):
+def test_logging__default_settings_logs_offline_dns(client: Client):
     """
-    :title: Default debug_level logs offline errors
+    :title: Default debug_level logs backend dns resolution issue
     :setup:
-        1. Configure SSSD with an invalid uri and enable ifp responder
+        1. Configure SSSD with an invalid uri and enable
         2. Start SSSD with default debug level
         3. Enable infopipe responder
     :steps:
         1. Check logs
         2. Check default domain status
     :expectedresults:
-        1. Logs contain connection errors
+        1. Logs contain dns error message
         2. SSSD is not connected
-    :customerscenario: True
+    :customerscenario: False
     """
     client.sssd.domain["ldap_uri"] = "ldap://typo.invalid"
     client.sssd.enable_responder("ifp")
     client.sssd.start(debug_level=None, raise_on_error=False)
 
     logs = client.fs.read(client.sssd.logs.domain())
-    assert "Failed to connect, going offline" in logs, "Offline error messages are not in logs!"
-
-    assert client.sssd.default_domain is not None, "Failed to load default domain!"
-    result = client.sssctl.domain_status(client.sssd.default_domain)
+    assert (
+        "Failed to resolve server 'typo.invalid': Domain name not found" in logs
+    ), "'Domain name not found' error message are not in logs!"
+    result = client.sssctl.domain_status(client.sssd.default_domain if client.sssd.default_domain else "test")
     assert result is not None
     assert "LDAP: not connected" in result.stdout, "LDAP is connected!"
 
 
 @pytest.mark.integration
 @pytest.mark.importance("low")
-@pytest.mark.ticket(bz=1416150)
 @pytest.mark.topology(KnownTopology.LDAP)
-def test_logging__default_settings_logs_to_syslog_when_ldap_is_offline(client: Client):
+def test_logging__default_settings_logs_offline_errors(client: Client, ldap: LDAP):
+    """
+    :title: Default debug_level logs offline errors
+    :setup:
+        1. Configure SSSD with ifp responder and LDAP backend
+        2. Reduce ldap_network_timeout to 1 second
+        3. Block LDAP access using firewall
+        4. Start SSSD with default debug level
+    :steps:
+        1. Check default domain status
+        2. Check domain log
+    :expectedresults:
+        1. SSSD is not connected
+        2. Logs contain connection/timout errors
+    :customerscenario: False
+    """
+    client.sssd.stop()
+    client.firewall.outbound.reject_host(ldap)
+    client.sssd.domain["ldap_network_timeout"] = "1"
+    client.sssd.enable_responder("ifp")
+    client.sssd.start(debug_level=None, raise_on_error=False)
+
+    time.sleep(60)
```

Comment:
Can this be lowered? ldap_network_timeout is 1, it looks like it can be.

Reply:
I tried 30 but it was flaky more failing than passing.

Reply:
How exactly did it fail?

Reply:
The offline messages were not written in logs.
Switched to polling loops instead of fixed sleeps.